README.md

```diff
@@ -3,31 +3,30 @@
 [![CircleCI](https://circleci.com/gh/HDE/arsenic/tree/master.svg?style=svg)](https://circleci.com/gh/HDE/arsenic/tree/master) [![Documentation Status](https://readthedocs.org/projects/arsenic/badge/?version=latest)](http://arsenic.readthedocs.io/en/latest/?badge=latest)
 
 
-Asynchronous, framework-independent webdriver client for AsyncIO/Aiohttp and
-Tornado.
+Asynchronous webdriver client built on asyncio.
 
 
 ## Quickstart
 
-Let's run Firefox via Geckodriver using Aiohttp.
+Let's run a local Firefox instance.
 
 
 ```python
```

Comment:
to avoid examples to be broken, one thing we could do is transfer this example in a python module that gets imported in Sphinx, and have the README.md point at RTD. 
OTHO having an example directly in the README is also cool, so I am not sure...